### ISSUE TITLE
Номер не является частью заголовка, он должен быть вне тега.

### ISSUE THREAD

**Issue by frontgirl** [Sonrisa86/diplomaNetology#3]
https://github.com/Sonrisa86/diplomaNetology/blob/97f62d2fa5b1dedda2af0e16f695488344fe9a9c/index.html#L53

**Comment by Sonrisa86** [comment]
В HTML вынесла, сломалось CSS немного. 
на мобильном все вернула, а планшет с десктопом пока не придумаю как поправить.
.athlets01, .athlets02, .athlets03 {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  padding-left: 46%;
  padding-right: 30px;
}
из-за этого свойства все в колонку встают, даже если h3 и span перебивают его на row, то все равно колонка.
h3 не хочет быть инлайном никак))

**Comment by frontgirl** [comment]
`<div class="heading-wrapper"><span class="heading-num">01</span><h3>...</h3></div> `

структура html должна получиться примерно такая. А дальше .heading-num должен быть абсолютно спозиционирован относительно header-wrapper.

По поводу flex-direction: column: ситуаций, когда flexbox нужен для вертикального расположения блоков, крайне мало. Вот и здесь мне видится, что можно обойтись без flexbox  вовсе, ведь простые блочные элементы div так и так располагаются друг под другом. А значит, flexbox здесь лишний.

**Comment by Sonrisa86** [comment]
вопрос: обязательно ли оборачивать в дополнительный wrapper если и так все встало на свои места?

**Comment by frontgirl** [comment]
Да, здесь дополнительный блок необходим, потому что сейчас если заголовок будет многострочный, будет так: https://tppr.me/XnB7B а для таких блоков принято следующее поведение: https://tppr.me/U7ee9
Вторая причина - показать с точки зрения семантики, что номер, хоть физически не является частью заголовка (h1-h6), но тем не менее, относится к оглавлению блока.

**Comment by Sonrisa86** [comment]
поняла, переделала, спасибо!
